gulpfile.js

```diff
@@ -0,0 +1,68 @@
+const gulp = require('gulp');
+const connect = require('gulp-connect');
+const open = require('gulp-open');
+const browserify = require('gulp-browserify');
+const rename = require('gulp-rename');
+const Server = require('karma').Server;
+
+const htmlSource = './index.html';
+const cssSource = './app/css/*.css';
+const jsSource = './app/controllers/controller.js';
+
+gulp.task('connect', () => {
+  connect.server({
+    root: ['./'],
+    port: 1337,
+    livereload: true,
+  });
+});
+
+gulp.task('open', () => {
+  gulp.src(__filename)
+  .pipe(open({ uri: 'http://localhost:1337' }));
+});
+
+gulp.task('html', () => {
+  gulp.src(htmlSource)
+  .pipe(connect.reload());
+});
+
+gulp.task('css', () => {
+  gulp.src(cssSource)
+  .pipe(connect.reload());
+});
+
+gulp.task('js', () => {
+  gulp.src(jsSource)
+  .pipe(connect.reload());
+});
```

Comment:
I feel task 'html', 'css' and 'js' can be combined into one task instead of separating them